governance/angle-dao.md

```diff
@@ -51,14 +51,16 @@ Angle Governance is represented by a 4-of-6 Gnosis multi-sig, whose role is to e
 
 It is composed of 6 person, three team members and three "public" crypto person, and requires a minimum of 4 signatures to execute a transaction. The multi-sig signers are:
 
-* Pablo Veyrat: [@pablo\_veyrat](https://twitter.com/pablo\_veyrat)
-* Guillaume Nervo: [@GuillaumeNervo](https://twitter.com/GuillaumeNervo)
-* Picodes: [@thePicodes](https://twitter.com/thePicodes)
-* Sébastien Derivaux: [@SebVentures](https://twitter.com/SebVentures)
-* Julien Bouteloup: [@bneiluj](https://twitter.com/bneiluj)
-* [0xMaki](https://twitter.com/0xMaki)
+- Pablo Veyrat: [@pablo_veyrat](https://twitter.com/pablo_veyrat)
+- Guillaume Nervo: [@GuillaumeNervo](https://twitter.com/GuillaumeNervo)
+- Picodes: [@thePicodes](https://twitter.com/thePicodes)
+- Sébastien Derivaux: [@SebVentures](https://twitter.com/SebVentures)
+- Julien Bouteloup: [@bneiluj](https://twitter.com/bneiluj)
+- [0xMaki](https://twitter.com/0xMaki)
 
-Multi-sig address: [0xdC4e6DFe07EFCa50a197DF15D9200883eF4Eb1c8](https://etherscan.io/address/0xdC4e6DFe07EFCa50a197DF15D9200883eF4Eb1c8)
+Multi-sig address on mainnet: [0xdC4e6DFe07EFCa50a197DF15D9200883eF4Eb1c8](https://etherscan.io/address/0xdC4e6DFe07EFCa50a197DF15D9200883eF4Eb1c8)
+
+There is one multisig per chain, each with the same signers. Their addresses on the other networks on can be found [here](https://developers.angle.money/overview/smart-contracts). 
```

Comment:
I'd add this paragraph below the: finalized votes paragraph. And specify before hand: Angle protocol is available on multiple different chains, there is one multisig per chain on which Angle is...